rpc/jsonrpc/server/http_server.go

```diff
@@ -146,6 +146,7 @@ func WriteRPCResponseHTTP(w http.ResponseWriter, res ...types.RPCResponse) error
 		return fmt.Errorf("json marshal: %w", err)
 	}
 	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Cache-Control", "max-age=31536000") // expired after one year
```

Comment:
I think there's some endpoints are not supposed to be cache, for example `/status`, the response will change according to current status.

Reply:
Yeah I think this is advisable. We should instead consider adding a cache policy on individual requests. Also, one year seems too large, no?

Reply:
@alexanderbez Agree with the cache policy that might need to be added. @yihuang, I will check the RPCs and add constrain to them.

Reply:
@alexanderbez Was my intuition, however, found a link to explain why set the age to one year if we see the data is immutable (but varnish cache doesn't support it)
https://ashton.codes/set-cache-control-max-age-1-year/

Reply:
Absolutely. If the response data does not change, then this makes sense 👍